Tasks/between.js

```diff
@@ -1,18 +1,22 @@
 // Extract substring between prefix and suffix
+'use strict';
 
-getvaluebetween = (str, p, s) => {
-  i = str.indexOf(p)
-    if ( i === -1 ) return ''
-    else {
-      k = i + p.length
-      str =  str.substring(k)
-      if ( s) {
-        i = str.indexOf( s )
-        if (i === -1) {
-          return  '' } else { str =  str.substring(0 , i)
-            }}}
-  return str
-}
+const getValueBetween = (str, fromLeft, fromRight) => {
+  let index = str.indexOf(fromLeft);
+  if (index === -1) {
+    return '';
+  } else {
```

Comment:
Use return-early principle and we don't need else here

Reply:
I thinked it will change the code style, but okay.